roles/horizon/tasks/main.yml

```diff
@@ -42,6 +42,18 @@
   notify:
     - reload apache
 
+- name: set horizon keystone policy fact
+  set_fact: policy_for_horizon=True
+  when: horizon.customize|default('False')|bool
+
+# Use custom policy to allow cloud_admin admin panel access
+- name: configure custom keystone policy
+  template: src=roles/keystone-defaults/templates/etc/keystone/policy.json dest={{ horizon.horizon_lib_dir }}/lib/python2.7/site-packages/openstack_dashboard/conf/keystone_policy.json }}
```

Comment:
In horizon `local_settings.py` we define a path for each policy file. We can define a path in /etc/ for the horizon version of the keystone policy file, instead of dropping it into the horizon python path.